docs/contributing.md

```diff
@@ -17,6 +17,14 @@ It is recommended that automated github actions are turned on in your fork to ha
 You can turn them on here:  
 ![Github actions example](./img/github-actions-example.png)
 
+* **When reviewing PRs, please leave a message what was done.**
+We don't have full test coverage, so manual testing can help.
+For reviews of PRs that could affect multiple worlds, it is helpful to state which games or settings were rolled.
+The original author may not have been able to test all worlds, or didn't know it would affect another world.
```

Comment:
I'll think about how to clarify the intentions better. As said on discord, manual testing may or may not be "required"* depending on how easy it would be to miss a code path in either automated or manual tests (by the author).

* can also test in beta/rc/production, but that has proven problematic

Reply:
Current revision addresses my concerns here